### PR TITLE
Add some Information

### PR DESCRIPTION
**Pull request by riegersan**
added:
- Applications
- Twitter Accounts
- Company Innoq
- Literature
- Podcasts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,37 @@ A list of awesome mobbing/ensemble/team programming resources. PRs are very welc
 ### Web
 
  - [mobti.me](https://mobti.me) - A free, collaborative, real-time mobbing timer.
+ - [mob.sh](https://mob.sh) - A free, script which gives you a simple set of shell commands for a fast handover.
+ - [timer.mob.sh](https://timer.mob.sh) - A timer, for the mob.sh script.
```
Comment (mrozbarry) [inline README.md]: Can you indent this and the following points, since they are part of the mob.sh ecosystem, please and thanks?

File: README.md [hunk 2]
```diff
@@ -7,17 +7,37 @@ A list of awesome mobbing/ensemble/team programming resources. PRs are very welc
...
 
+## Literature
+
+ - [Mob Programming: A whole Team approach](https://leanpub.com/mobprogramming) - Great book about the beginning of Mob Programming
+ - [Remote Mob Programming](https://leanpub.com/remotemobprogramming) - A short book about remote Mob Programming
+ - [Code with the Wisdom of the Crowd: Get Better Together with Mob Programming](https://amzn.to/3JK0vt9) - A short book about Mob Programming
+
+## Podcasts
+
+### Mob Mentality Show
```
Comment (mrozbarry) [inline README.md]: Let's turn this into

- Mob Mentality Show
  - Apple
  - Google
  - Spotify
  - Amazon

**Review by mrozbarry — CHANGES_REQUESTED** [review]
Thanks for adding to the awesome list! Just a few little edits, then we're good to go. Thanks a lot!

File: README.md (hunk 1 above)
Comment (riegersan) [reply README.md]: makes sense

File: README.md (hunk 2 above)
Comment (riegersan) [reply README.md]: looks better than my approach. I will change it

**Comment by riegersan** [comment]
all requested changes are implemented with commit 8ab9326

**Pull request merged**